src/app/courses/courses.component.ts

```diff
@@ -0,0 +1,25 @@
+import { CoursesService } from './../shared/courses/courses.service';
+import { Course } from '../shared/courses/course.interface';
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-courses',
+  templateUrl: './courses.component.html',
+  styleUrls: ['./courses.component.scss']
+})
+export class CoursesComponent implements OnInit {
+
+  public courses: Course[];
```

Comment:
by default all properties without `private`/`protected` are public, so it is unnecessary to use `public` modifier here and in other places.

Reply:
done